shopinvader_product_attribute_set/schemas/product_attribute.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from __future__ import annotations
+
+from enum import Enum
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+from odoo.addons.product.models.product_product import ProductProduct
+
+from ..models.attribute_attribute import AttributeAttribute
+
+
+class ProductAttributeType(Enum):
+    """Enum for product attribute type"""
+
+    char = "char"
+    text = "text"
+    select = "select"
+    multiselect = "multiselect"
+    boolean = "boolean"
+    integer = "integer"
```

Comment:
float is missing here

Reply:
Fixed it here https://github.com/shopinvader/odoo-shopinvader/pull/1491
Feel free to fix here or to merge the other one. 
I just needed a complete PR to test ;)

Reply:
Thank you @simahawk !